explorer/lib/explorer_web/live/pages/batches/index.html.heex

```diff
@@ -1,5 +1,11 @@
 <div class="flex flex-col space-y-3 text-foreground px-1 sm:max-w-lg md:max-w-3xl lg:max-w-5xl mx-auto capitalize">
   <.card_preheding>Batches</.card_preheding>
+  <!-- Progress bar based on https://dev.to/mikekennedydev/progress-bar-text-colour-1gg6 -->
+  <.live_component
+  id="labeled-progress-bar"
+  module={LabeledProgressBarComponent}
+  percent_progress={@next_batch_countdown_progress}
+  label={"Next batch in #{@time_to_next_block} minutes"}/>
```

Comment:
when time_to_next_block==0 , another message would be nice
`next batch in any moment now!`